glide.lock

```diff
@@ -127,13 +127,12 @@ imports:
 - name: github.com/projectcalico/go-yaml-wrapper
   version: 598e54215bee41a19677faa4f0c32acd2a87eb56
 - name: github.com/projectcalico/libcalico-go
-  version: e661addd9596b291db14c489f6e56575d493982f
+  version: 8953ccf223be94fd0ea931a66df15b3f1ba3f054
```

Comment:
that version of libcalico-go is 3 or 4 behind head. intentional?

Reply:
hmm that's odd - i thought i'd only just done a glide up

Reply:
i'll redo

Reply:
@gunjan5 pointed out that glide.yaml "master" doesn't actually work as expected, but rather pulls the cached version. Suggest you either ping to a git commit hash or elide the `version` entirely.

Reply:
yeah that suddenly dawned on me.

Reply:
my personal suggestion is `elide the version entirely.` whatever that fancy word means

Reply:
`elide` means "I went to community college."